Buildings/Controls/OBC/ASHRAE/G36_PR1/Generic/SetPoints/OperationMode.mo

```diff
@@ -38,12 +38,12 @@ block OperationMode "Block that outputs the operation mode"
     final quantity="Time") "Maximum warm-up time among all the zones"
     annotation (Placement(transformation(extent={{-300,130},{-260,170}}),
         iconTransformation(extent={{-140,40},{-100,80}})));
-  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput occHeaHigMin
-    "True when the occupied heating setpoint temperature is higher than the minimum zone temperature"
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uOccHeaHig
+    "True when the occupied heating setpoint temperature is higher than the zone temperature"
```

Comment:
I would want it to keep as before. This would be the zone temperature (for the single zone VAV system). 

But for the multizone system, it should be the minimum zone temperature. If it is `false`, it means that the minimum zone temperature is higher than the occupied heating setpoint (all the zone temperature have been above the occupied heating setpoint), the warm-up mode should be finished (although it may still be in the possible warm-up period).

Reply:
This issue is related to setback/setup mode. If occupied/unoccupied setpoints are different from one zone to another, is it still valid to use TZonMax or TZonMin to compare with zone temperatures across different zones? I imagined the block ZoneStatus will be instantiated once for each zone. Please let me know what you think.